repo2docker/buildpacks/python/requirements.frozen.txt

```diff
@@ -0,0 +1,41 @@
+# AUTO GENERATED FROM requirements.txt, DO NOT MANUALLY MODIFY
+# Frozen on Thu Nov 30 08:51:37 UTC 2017
+bleach==2.1.1
+decorator==4.1.2
+entrypoints==0.2.3
+html5lib==1.0b10
+ipykernel==4.6.1
+ipython==6.2.1
+ipython-genutils==0.2.0
+ipywidgets==6.0.0
+jedi==0.11.0
+Jinja2==2.10
+jsonschema==2.6.0
+jupyter-client==5.1.0
+jupyter-core==4.4.0
+jupyterlab==0.28.0
+jupyterlab-launcher==0.6.0
```

Comment:
This is problematic :). It should be `<0.6.0` if you're using jupyterlab 0.28.0.

Reply:
(See #154)

Reply:
@jasongrout if you look at 'freeze.bash' you can see the script that's generating these frozen files. Can you tell me what's wrong with it?

Reply:
@jasongrout I've updated to JupyterLab 0.29.2 and maybe the requirements look better now?

Reply:
yes, that would be better.